src/components/SocialVolumeWidget/socialVolumeEpic.js

```diff
@@ -1,17 +1,10 @@
 import { Observable } from 'rxjs'
 import moment from 'moment'
 import * as actions from './actions'
-import {
-  socialVolumeGQL,
-  totalSocialVolumeGQL,
-  wordTrendScoreGQL,
-  allProjetsGQL
-} from './socialVolumeGQL'
+import { socialVolumeGQL } from './socialVolumeGQL'
```

Comment:
Let's try to use SOCIAL_VOLUME_QUERY name, as Apollo team does. I used it for new code and tried to refactor old. @DmitriVanGuard